### PR TITLE
#162948887 Users should be able to see time it takes to read an article

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What does this PR do?
* Implement read time on articles

#### Description of Task to be completed?
* Enable users to see the time it takes to read an article

#### How should this be manually tested?
* Clone the github repository `git clone https://github.com/andela/ah-alpha-frontend` on your local machine.
* run `git checkout ft-readtime-162948887`
* Install the required dependencies using `npm install`
* Run the local development server using `npm start`

#### Screenshot (on viewing all articles)
image

#### Screenshot (on viewing a single article)
image

#### What are the relevant pivotal tracker stories?
[#162948887](https://www.pivotaltracker.com/story/show/162948887)

### CODE REVIEW

File: src/components/containers/articles/Articles.js [hunk 1]
```diff
@@ -106,6 +106,9 @@ export class GetArticles extends Component {
                                     "YYYYMMDD"
                                   ).fromNow()}
                                 </span>
+                                <br />
+                                <Icon name="clock" size="large" />
+                                <span className="read">{article.read_time.substr(3, 1)} min read</span>
```
Comment (tesh254) [inline src/components/containers/articles/Articles.js]: Format this to take up multiple lines

File: src/components/containers/articles/OneArticle.js [hunk 2]
```diff
@@ -96,7 +96,10 @@ export class GetOneArticle extends Component {
                   <Icon name="star" size="large" className="star-icon" />
                   {article.rating ? parseFloat(article.rating).toFixed(1) : <span />}
                   <br />
-                  <span className="read-time"> {moment(article.created_at).format("MMM Do")}</span>
+                  <span className="read-time"> {moment(article.created_at).format("MMM Do")}</span><br />
```
Comment (tesh254) [inline src/components/containers/articles/OneArticle.js]: Also this anyway it looks good

**Review by tesh254 — APPROVED** [review]

**Review by boltonotieno — CHANGES_REQUESTED** [review]
Nice on work on the implementation. However, from your screenshots,  the same way you've displayed the read time on the articles card `2 min read `, you could do the same on the single article page instead of `2 min` without the `read ` word

**Comment by Ruiru11** [comment]
> Nice on work on the implementation. However, from your screenshots, the same way you've displayed the read time on the articles card `2 min read `, you could do the same on the single article page instead of `2 min` without the `read ` word

(the fast implementation was as you are requesting but the PO said we should do away with the `read`)

**Comment by boltonotieno** [comment]
> > Nice on work on the implementation. However, from your screenshots, the same way you've displayed the read time on the articles card `2 min read `, you could do the same on the single article page instead of `2 min` without the `read ` word
> 
> (the fast implementation was as you are requesting but the PO said we should do away with the `read`)

Ooh, that's okay.

**Review by boltonotieno — APPROVED** [review]

File: src/components/containers/articles/OneArticle.js [hunk 3]
```diff
@@ -96,7 +97,14 @@ export class GetOneArticle extends Component {
                   <Icon name="star" size="large" className="star-icon" />
                   {article.rating ? parseFloat(article.rating).toFixed(1) : <span />}
                   <br />
-                  <span className="read-time"> {moment(article.created_at).format("MMM Do")}</span>
+                  <br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (danuluma) [inline src/components/containers/articles/OneArticle.js]: This could be padded better using css

**Review by danuluma — APPROVED** [review]
Nice work

**Pull request merged**